internal/subproviders/morpheus/resources/instance/instance.go

```diff
@@ -162,6 +162,10 @@ func getInstanceAsState(
 	// config
 	state.Config = types.DynamicNull()
 
+	if !plan.Config.IsNull() && !plan.Config.IsUnknown() {
```

Comment:
Not mentioned in commit.
If you're doing things other than docs, should be in seperate pr, or mentioned in commit.